monofony/front-bundle/0.1/manifest.json

```diff
@@ -0,0 +1,13 @@
+{
+    "bundles": {
+        "Monofony\\Bundle\\FrontBundle\\MonofonyFrontBundle": ["all"]
```

Comment:
This does also look like "bundle" only made for copying data to the users application.

Reply:
There's a dependency injection configuration:
https://github.com/Monofony/FrontBundle/blob/master/DependencyInjection/MonofonyFrontBundleExtension.php